docs/reference/edgectl-download.md

```diff
@@ -0,0 +1,42 @@
+# Downloads
+
+Edge Control is available for macOS, Linux, and Windows.
+
+Ensure that you have 
+
+## macOS
+
+1. To download by curl command, run the following in your command line:
+
+    ```shell
+    sudo curl -fL https://metriton.datawire.io/downloads/darwin/edgectl -o /usr/local/bin/edgectl && sudo chmod a+x /usr/local/bin/edgectl
+    ```
+
+2. Or, download the [binary file](https://metriton.datawire.io/downloads/darwin/edgectl).
```

Comment:
2-4 are not in sequence with 1. What I mean is that the instructions are "do 1 then 5-6" or "do 2-4 then 5-6". So the way it is written right now is confusing to the reader.